crates/ide_db/src/tests/sourcegen_lints.rs

```diff
@@ -173,19 +190,28 @@ fn generate_descriptor_clippy(buf: &mut String, path: &Path) {
     for clippy_lint in clippy_lints.into_iter() {
         let lint_ident = format!("clippy::{}", clippy_lint.id);
         let doc = clippy_lint.help;
-        push_lint_completion(buf, &lint_ident, &doc);
+        push_lint_completion(buf, &lint_ident, &doc, vec![]);
     }
     buf.push_str("];\n");
 }
 
-fn push_lint_completion(buf: &mut String, label: &str, description: &str) {
+fn push_lint_completion<'a>(
+    buf: &mut String,
+    label: &str,
+    description: &str,
+    children: Vec<String>,
```

Comment:
I tried to make this work with `Iterator<Item=String>` and making it optional, but to me it gave no benefit.